legacy/testdata/migrations/rulesets.json

```diff
@@ -218,7 +218,7 @@
                 {
                     "category": {"eng": "< 10"}, 
                     "uuid": "1c75fd71-027b-40e8-a819-151a0f8140e6", 
-                    "destination": "7d40faea-723b-473d-8999-59fb7d3c3ca2", 
+                    "destination": "5b977652-91e3-48be-8e86-7c8094b4aa8f", 
```

Comment:
now that loading legacy flows uses same validation rules - these actually have to be existing nodes